testing/sdk_install.py

```diff
@@ -41,14 +41,15 @@ def install(
         timeout_sec=timeout_seconds,
         expected_running_tasks=running_task_count)
 
+    # Regardless of wait_scheduler_idle, it's safe to wait for the deployment from an install to complete
+    # before proceeding. This can take a while, default is 15 minutes. for example with HDFS, we can hit
+    # the expected total task count via FINISHED tasks, without actually completing deployment.
+    sdk_utils.out("Waiting for {}/{} to finish deployment plan...".format(package_name, service_name))
+    sdk_plan.wait_for_completed_deployment(service_name, timeout_seconds)
```

Comment:
Do we not test deployments that fail? In that case, would the completed test pass?